src/main/java/com/dev/Main.java

```diff
@@ -0,0 +1,12 @@
+package com.dev;
+
+public class Main {
+    public static void main(String[] args) {
+        Counter counter = new Counter();
+        counter.setCounter(0);
+        MyThread myThread = new MyThread(counter);
```

Comment:
```suggestion
        Counter counter = new Counter();
        MyThread myThread = new MyThread(counter);
```
it's primitive it'll be 0 by default

Reply:
of course, fixed)